docs/autosummary.rst

```diff
@@ -0,0 +1,33 @@
+.. Automatically generates summaries and documentation.
+
+.. automodule:: qctrlopencontrols
+  .. autosummary::
+    :toctree: .
+    :template: drivencontrol/drivencontrol.rst
+
+    driven_controls
+
+.. automodule:: qctrlopencontrols
+  .. autosummary::
+    :toctree: .
+    :template: drivencontrol/drivencontrol.rst
```

Comment:
If we're using this template here then it should be renamed.

Reply:
changed.